### DISCUSSION TITLE
How to update http version on linux red hat 7.6 in CLI?

### DISCUSSION THREAD

**Discussion by WizardKali** [community/community#38431] [Other Feature Feedback, Questions, & Ideas]
How to update http version with terminal command?
Wireshark did show me, http ver is 1.1, thats why web pages not work.

**Comment by airtower-luna** [comment]
You don't, the HTTP client chooses the protocol version it wants to use. Some clients have flags to configure which HTTP version you want to use. But if a website doesn't support HTTP/1.1 it's pretty severely broken.

**Reply by WizardKali** [reply]
How to update this? Somehow

**Reply by airtower-luna** [reply]
You'll first have to figure out where the actual problem is. As I said, using HTTP/1.1 is not a bug as such.

**Reply by WizardKali** [reply]
Thank you!
